MODULE.bazel

```diff
@@ -0,0 +1,29 @@
+module(
+    name = "googleapis",
+    version = "0",
+    repo_name = "com_google_googleapis",
+)
+
+bazel_dep(name = "grpc", version = "1.56.3.bcr.1", repo_name = "com_github_grpc_grpc")
+bazel_dep(name = "grpc-java", version = "1.62.2", repo_name = "io_grpc_grpc_java")
+bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
+bazel_dep(name = "rules_go", version = "0.46.0", repo_name = "io_bazel_rules_go")
+bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
+bazel_dep(name = "rules_python", version = "0.31.0")
+
+switched_rules = use_extension("//:extensions.bzl", "switched_rules")
+
+# TODO: Enable support for other languages with bzlmod
+switched_rules.use_languages(
+    cc = True,
+    # csharp = True,
+    # gapic = True,
```

Comment:
i left some of these out, which means not everything in the repo builds with bzlmod enabled, in order to support each of these we'll have to move their deps to bzlmod, which is potentially difficult depending on their transitives